.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_USER
+# DOCKER_PASS
+#
+
+version: 2
+jobs:
+  build:
+    machine:
+      enable: true
+    working_directory: ~/mozilla/telemetry-analysis-service
```

Comment:
I think you can abstract the working_directory away in a global config if all jobs have the same.